src/tests/unit/services/SendEmailService.test.ts

```diff
@@ -179,22 +189,7 @@ describe("SendEmailProcessor", () => {
 		const eventBody = JSON.parse(sqsEvent.Records[0].body);
 		const email = Email.parseRequest(JSON.stringify(eventBody.Message), logger);
 		await expect(sendEmailServiceTest.sendYotiPdfEmail(email)).rejects.toThrow();
-		expect(mockGovNotify.sendEmail).toHaveBeenCalledTimes(4);
-	});
-
-	it("Write to txMA fails when session id not found in the DB", async () => {
```

Comment:
Why have we removed this test?

Reply:
We have this in the test 
mockF2fService.getSessionById.mockResolvedValue(undefined);

Given latest code changes if getSessionById is undefined code will never reach the TxMA stage as it'll error out before then so this test is now invalid